src/image/VisitInfo.cc

```diff
@@ -414,6 +414,15 @@ geom::Angle VisitInfo::getLocalEra() const { return getEra() + getObservatory().
 
 geom::Angle VisitInfo::getBoresightHourAngle() const { return getLocalEra() - getBoresightRaDec()[0]; }
 
+geom::Angle VisitInfo::getBoresightParAngle() const {
+  double _parallactic_y, _parallactic_x, result;
+  _parallactic_y = sin(getBoresightHourAngle());
```

Comment:
Please make the conversion to radians explicit, e.g. `getBoresightHourAngle.asRadians()`